source/release-notes/1.6.txt

```diff
@@ -42,6 +42,50 @@ For more information, see :ref:`c2c-filter-regex`.
 
 Starting in 1.6.0, ``mongosync`` supports sync with MongoDB 7.0.
 
+Other Notes
+~~~~~~~~~~~
+
+New Features:
+
+- Sped up the commit process by parallelizing metadata cleanup. 
+  (:issue:`REP-2568`)
+
+- Reduced memory allocation during change event application by ~30% for 
+  pre-6.0 releases of MongoDB Server versions. (:issue:`REP-3015`)
```

Comment:
Could we remove all jira rep tickets here? We only included them in the release notes to make it easier for us to reference them. Apologies for the confusion!

Reply:
Fixed